how-to/workspace-platform-starter/client/src/framework/workspace/notification-client.ts

```diff
@@ -200,7 +204,15 @@ export class NotificationClient implements NotificationClientInterface {
 		if (!isEmpty(mappedListener)) {
 			await notificationRemoveEventListener(eventType as never, mappedListener.wrappedListener);
 			this._listenerRegister = this._listenerRegister.filter((entry) => entry.id !== mappedListener.id);
-		}
+            if(eventType === "notifications-count-changed") {
+                const countListeners = this._listenerRegister.filter((entry) => entry.eventType === eventType);
+                if(countListeners.length === 1) {
```

Comment:
Shouldnt this be 0

Reply:
The reason it's one is because we add our own listener. So even if all subscribers removed themselves there would be the initial one we added. That's why when it gets down to 1 we unsubscribe the first entry as well and set the tracked flag back to false (until someone else is interested in count).